cmd/kubeless/update.go

```diff
@@ -72,9 +73,14 @@ var updateCmd = &cobra.Command{
 		if err != nil {
 			logrus.Fatal(err)
 		}
+		funcMem := resource.Quantity{}
+		if mem != "" {
+			funcMem = parseMemory(mem)
+		}
 
 		funcType := "HTTP"
-		err = utils.UpdateK8sCustomResource(runtime, handler, file, funcName, funcType, ns, description, mem, labels, envs)
+		f := constructFunction(runtime, handler, file, funcName, funcType, "", ns, "", description, funcMem, labels, envs)
```

Comment:
>  I suggest you create the actual Function object right here, inline

But it will be duplicated code where we construct Function object at both `deploy` and `update` cmd. I made `constructFunction` for that reason. But I agree we don't want to have a function that requires a long list of arguments.